app/api/utility.py

```diff
@@ -451,54 +447,25 @@ def load_json(path: Path) -> dict:
         return json.load(f)
 
 
-def fetch_and_save_cogatlas(output_path: Path):
+def create_snomed_assessment_lookup(output_path: Path):
     """
-    Fetches the Cognitive Atlas vocabulary using its native Task API and writes term ID-label mappings to a temporary lookup file.
-    If the API request fails, a backup copy of the vocabulary is used instead.
+    Reads in a file of assessment terms from the SNOMED vocabulary and writes term ID-label mappings to a temporary lookup file.
 
-    Saves a JSON with keys corresponding to Cognitive Atlas task IDs and values corresponding to human-readable task names.
+    Saves a JSON with keys corresponding to SNOMED IDs and values corresponding to human-readable term names.
 
     Parameters
     ----------
     output_path : Path
         File path to store output vocabulary lookup file.
     """
-    api_url = "https://www.cognitiveatlas.org/api/v-alpha/task?format=json"
-
-    try:
-        response = httpx.get(url=api_url)
-        if response.is_success:
-            vocab = response.json()
-        else:
-            warnings.warn(
-                f"""
-                The API was unable to fetch the Cognitive Atlas task vocabulary (https://www.cognitiveatlas.org/tasks/a/) from the source and will default to using a local backup copy of the vocabulary instead.
-
-                Details of the response from the source:
-                Status code {response.status_code}
-                {response.reason_phrase}: {response.text}
-                """
-            )
-            # Use backup copy of the raw vocabulary JSON
-            vocab = load_json(BACKUP_VOCAB_DIR / "cogatlas_task.json")
-    except httpx.NetworkError as exc:
-        warnings.warn(
-            f""""
-            Fetching of the Cognitive Atlas task vocabulary (https://www.cognitiveatlas.org/tasks/a/) from the source failed due to a network error.
-            The API will default to using a local backup copy of the vocabulary instead.
-
-            Error: {exc}
-            """
-        )
-        # Use backup copy of the raw vocabulary JSON
-        vocab = load_json(BACKUP_VOCAB_DIR / "cogatlas_task.json")
+    vocab = load_json(BACKUP_VOCAB_DIR / "snomed_assessment.json")
 
-    term_labels = {term["id"]: term["name"] for term in vocab}
+    term_labels = {term["identifier"][7:]: term["label"] for term in vocab}
```

Comment:
this double index confuses my brain. On a related note: it seems that we now have two snomed sub-vocabularies with different formats: 

**snomed disorder**:
```
{
    "sctid": "10007009",
    "preferred_name": "Coffin-Siris syndrome"
  },
```
and **snomed assessment**:

```
[
  {
    "identifier": "snomed:1231439001",
    "label": "Calgary Depression Scale for Schizophrenia"
  },
```
I think we should make both consistent and I think the assessment shape (identifier + label) is nicer

Reply:
yeah, this discrepancy is related to the issue we discovered right before the break, where the vocab file formats currently used by the API and annotation tool are different. Specifically, the API (on `main`) uses vocab files that do not hardcode any vocab namespace prefixes

Here's the relevant issue https://github.com/neurobagel/planning/issues/216. I think the important decision would be whether we want to hardcode the prefixes or not into these files (see the "Considerations" section of the issue).

Reply:
Also, agree with the double index being a bit hard to read! How about:
```suggestion
    term_labels = {term["identifier"].removeprefix("snomed:"): term["label"] for term in vocab}
```
should work for 3.9+

Reply:
Do we want to switch to identifier + label as part of this PR?

Reply:
I would address that as part of https://github.com/neurobagel/planning/issues/216, since I think the format would benefit from some discussion